benchling/src/main/java/com/decibeltx/studytracker/benchling/eln/BenchlingElnRestClient.java

```diff
@@ -217,7 +217,7 @@ public BenchlingFolder createFolder(String name, String parentFolderId) {
 
   @Override
   public BenchlingEntry createEntry(BenchlingEntryRequest entryRequest) {
-    URL url = joinUrls(rootUrl, "/api/v2/folders");
+    URL url = joinUrls(rootUrl, "/api/v2/entries");
```

Comment:
I think it makes sense to extract constant

Reply:
they have this kind of style applied throughout the code.

Reply:
Ok